webhooks/tasks.py

```diff
@@ -0,0 +1,71 @@
+"""
+Celery tasks.
+"""
+import hashlib
+import hmac
+import json
+import logging
+
+from django.apps import apps
+from django.utils.encoding import force_bytes
+import requests
+
+from ccxcon.celery import async
+from webhooks.models import Webhook
+
+log = logging.getLogger(__name__)
+
+
+@async.task
+def publish_webhook(model_str, pk):
+    """
+    Publishes model updates to various services as defined by Webhook instances.
+    """
+    try:
+        app, model_label = model_str.split('.')
+    except ValueError:
+        log.warning("Malformed model string for webhook serialization: %s",
```

Comment:
I believe if you raise an exception in a task, celery will add that result to the result store. We won't ever check the result store for this, so we instead just log the error and provide no return value.